hedera-mirror-rest/__tests__/specs/contracts/results/{id}/ethereum-transaction-contract-create-with-empty-to-address.json

```
@@ -0,0 +1,105 @@
+{
+  "description": "Contract results api call using transactionId or an ethereum transaction hash for contract creation with empty to address in ethereum transaction table",
+  "setup": {
+    "entities": [
+      {
+        "num": 5001,
+        "evm_address": "0x867623472bd7c14e5b5ad0b17f7e810490d07718"
+      }
+    ],
+    "contractresults": [
+      {
+        "amount": 30,
+        "bloom": [5, 5],
+        "call_result": [6, 6],
+        "consensus_timestamp": "167654000123456",
+        "contract_id": 5001,
+        "created_contract_ids": [8001],
+        "error_message": "",
+        "function_parameters": [],
+        "function_result": [],
+        "gas_limit": 1000000,
+        "gas_used": 123,
+        "payer_account_id": 8001,
+        "sender_id": 8001,
+        "transaction_result": 22,
+        "transaction_hash": "4a563af33c4871b51a8b108aa2fe1dd5280a30dfb7236170ae5e5e7957eb6392",
+        "transaction_index": 1,
+        "transaction_nonce": 0,
+        "evmAddress": "867623472bd7c14e5b5ad0b17f7e810490d07718"
+      }
+    ],
+    "recordFiles": [
+      {
+        "index": 17,
+        "gas_used": 50000000,
+        "consensus_start": 167654000123450,
+        "consensus_end": 167654000123460,
+        "hash": "6ceecd8bb224da491"
+      }
+    ],
+    "transactions": [
+      {
+        "consensus_timestamp": 167654000123456,
+        "payerAccountId": "8001",
+        "transaction_hash": "519a008fabde4d",
+        "type": 50,
+        "valid_start_timestamp": 167654000123400
+      }
+    ],
+    "ethereumtransactions": [
+      {
+        "call_data": "0x0707",
+        "consensus_timestamp": 167654000123456,
+        "hash": "4a563af33c4871b51a8b108aa2fe1dd5280a30dfb7236170ae5e5e7957eb6392",
+        "payer_account_id": "8001",
+        "value": "0x0077359400",
+        "_value_description": "2000000000 tinybars, in importer java BigInteger.toByteArray always adds extra 0x00",
+        "to_address": []
```

q: should this be null?
```suggestion
        "to_address": null
```

---

I asked the same question but you were first :racing_car: .